### DISCUSSION TITLE
Manual execution of X profiles one by one

### DISCUSSION THREAD

**Discussion by szpak** [borgbase/vorta#2150] [FAQ]
Hi. I have a few profiles for different "directories" (types of data) backing up to the same Borg repository (for deduplication), but with different data retention. I start it manually. I wonder, if there is a way to "schedule" X different profiles for manual executions to be performed one by one (i.e. I run the first profile and after its success, the others are automatically called). The backup is performed over a slow transport and it may take some time, so I would prefer not to have to wait and check often if the the first finished to execute the second (and so on).

**Comment by m3nu** [comment]
There is not currently, but sounds like an interesting use case. Maybe by selecting them on the right and then doing "Start backup"?

**Reply by szpak** [reply]
Sounds good.

Any errors should interrupt the process, but after successful execution some simple summary could be displayed.

Btw, do you an ability to convert it into issue (feature request) or I should create a new one there?
